editor/plugins/gizmos/decal_gizmo_plugin.cpp

```diff
@@ -38,13 +38,7 @@
 
 DecalGizmoPlugin::DecalGizmoPlugin() {
 	helper.instantiate();
-	Color gizmo_color = EDITOR_GET("editors/3d_gizmos/gizmo_colors/decal");
-
-	create_icon_material("decal_icon", EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("GizmoDecal"), EditorStringName(EditorIcons)));
-
-	create_material("decal_material", gizmo_color);
-
-	create_handle_material("handles");
+	DecalGizmoPlugin::update_materials();
```

Comment:
The class qualifier is redundant as far as I can tell:

```suggestion
	update_materials();
```

Reply:
Rider was upset because we're calling a virtual function within the constructor. 

Following other classes within godot:
https://github.com/godotengine/godot/blob/f1c0b5b8542afb750236b8d35d533737b883de29/editor/gui/editor_file_dialog.cpp#L179-L180